packages/mobile/src/transactions/send.ts

```diff
@@ -25,7 +24,7 @@ const TAG = 'transactions/send'
 // causing failures when a tx times out (rare but can happen on slow devices)
 const TX_NUM_TRIES = 1 // Try txs up to this many times
 const TX_RETRY_DELAY = 2000 // 2s
-const TX_TIMEOUT = 40000 // 40s
+const TX_TIMEOUT = 45000 // 45s
```

Comment:
Why increase tx timeout when nonces are no longer manual?

Reply:
Just a precaution since Ankit also saw a case where the tx timed out